traitsui/testing/tester/qt4/helpers.py

```diff
@@ -110,6 +111,30 @@ def key_sequence_qwidget(control, interaction, delay):
     QTest.keyClicks(control, interaction.sequence, delay=delay)
 
 
+def key_sequence_textbox(control, interaction, delay):
+    """ Performs simulated typing of a sequence of keys on a widget that is
+    a textbox. The keys are restricted to values also supported for testing
+    wx.TextCtrl.
+
+    Parameters
+    ----------
+    control : QWidget
+        The Qt widget intended to hold text for editing.
+        e.g. QLineEdit and QTextEdit
+    interaction : instance of command.KeySequence
+        The interaction object holding the sequence of key inputs
+        to be simulated being typed
+    delay : int
+        Time delay (in ms) in which each key click in the sequence will be
+        performed.
+    """
+    for key in interaction.sequence:
+        check_key_compat(key)
```

Comment:
For future reference, if we did not check and try to use QTest with a character e.g. "ಶ", then Qt will throw this error:
```
ASSERT: "false" in file /Users/builder/workspace/Buildsystem/test/pisi/tmp/Qt-5.12.6-5/work/qt/qtbase/src/testlib/qasciikey.cpp, line 228
Abort trap: 6
```
If there are needs to support a wider range of characters, then we would need to do something else. Depending on the use case, we may manually edit the text and fire the appropriate Qt event used by the editor.

Reply:
The test `test_key_sequence_unsupported_key` demonstrates the above error if the check was not added.